tests/v3_api/common.py

```diff
@@ -221,6 +221,16 @@ def validate_workload_with_sidekicks(p_client, workload, type , ns_name, pod_cou
         assert "running" in pod["status"]["containerStatuses"][1]["state"]
 
 
+def validate_workload_image(p_client, workload, expectedImage):
+    image = p_client.list_workload(workload=workload.id)[0].containers[0].image
+    assert image == expectedImage
+
```

Comment:
@adingilloRancher Include the check for making sure that the pods of the workloads have been started with right image using kubectl command.